files/ko/web/css/border/index.html

```diff
@@ -116,36 +115,17 @@ <h4 id="결과">결과</h4>
 
 <h2 id="Specifications" name="Specifications">명세</h2>
 
-<table class="standard-table">
- <thead>
-  <tr>
-   <th scope="col">Specification</th>
-   <th scope="col">Status</th>
-   <th scope="col">Comment</th>
-  </tr>
- </thead>
- <tbody>
-  <tr>
-   <td>{{SpecName('CSS3 Backgrounds', '#the-border-shorthands', 'border')}}</td>
-   <td>{{Spec2('CSS3 Backgrounds')}}</td>
-   <td>Removes <em>specific</em> support for <code>transparent</code>, as it is now a valid {{cssxref("&lt;color&gt;")}}; this has no practical impact.<br>
-    Though it cannot be set to a custom value using the shorthand, <code>border</code> now resets {{cssxref("border-image")}} to its initial value (<code>none</code>).</td>
-  </tr>
-  <tr>
-   <td>{{SpecName('CSS2.1', 'box.html#border-shorthand-properties', 'border')}}</td>
-   <td>{{Spec2('CSS2.1')}}</td>
-   <td>Accepts the <code>inherit</code> keyword. Also accepts <code>transparent</code> as a valid color.</td>
-  </tr>
-  <tr>
-   <td>{{SpecName('CSS1', '#border', 'border')}}</td>
-   <td>{{Spec2('CSS1')}}</td>
-   <td>Initial definition.</td>
-  </tr>
- </tbody>
-</table>
+{{Specifications}}
 
 <h2 id="Browser_compatibility" name="Browser_compatibility">브라우저 호환성</h2>
 
-
-
 <p>{{Compat("css.properties.border")}}</p>
+
+<h2 id="see_also">같이 보기</h2>
+<ul>
+  <li>{{Cssxref("border-width")}}</li>
+  <li>{{Cssxref("border-style")}}</li>
+  <li>{{Cssxref("border-color")}}</li>
+  <li><a href="/ko/docs/Web/CSS/CSS_Backgrounds_and_Borders">배경 및 테두리</a></li>
+  <li><a href="/ko/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders">css 배우기: 배경 및 테두리</a></li>
```

Comment:
```suggestion
  <li><a href="/ko/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders">CSS 배우기: 배경 및 테두리</a></li>
```
CSS는 고유명칭이기 때문에 대문자를 사용해주세요.

Reply:
확인해주셔서 감사합니다! 다음에는 더욱 꼼꼼히 작업해야겠어요🥲